package.json

```diff
@@ -107,7 +107,6 @@
     "lint:js": "eslint .",
     "lint:css": "stylelint --fix 'public/css/**/*.css'",
     "lint:audit": "npm audit",
-    "pretest": "npm run lint",
```

Comment:
Removing this to run `npm run lint` as an explicit parallel build job on Travis.